src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY_OPERATION_TYPE = OperationType.SUPPLY.name().toLowerCase();
+    private static final String BUY_OPERATION_TYPE = OperationType.BUY.name().toLowerCase();
+    private static StringBuilder result;
+    private static String textFromFile;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        createReport();
+        writeReportToFile(toFileName);
```

Comment:
Try to avoid **void** methods 
```suggestion
        String textFromFile = readFromFile(fromFileName);
        String report = createReport();
        writeReportToFile(toFileName, report);
```